target/linux/rtl838x/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+
```

Comment:
This double empty line could be removed if touched again.

Reply:
Will be fixed.